### PR TITLE
Fix tutorial

### PR DESCRIPTION
**Pull request by Sam-Tygier**
Fix issues in the tutorials and examples.

### CODE REVIEW

File: ExamplesTutorials/Tutorials/7-CollimationAndScattering/CollimationAndScattering.cpp [hunk 1]
```diff
@@ -232,8 +232,7 @@ int main()
 	delete dispersion;
 	delete particleBunch;
 	delete tracker;
-	// delete scatterModel;
-	delete collimateProcess;
```
Comment (scottrowan) [inline ExamplesTutorials/Tutorials/7-CollimationAndScattering/CollimationAndScattering.cpp:236]: did you mean to remove `delete collimateProcess`? it is initialized with a 'new' on line 194?

**Review by scottrowan — APPROVED** [review]
Looks good, approved pending confirmation of 'delete collimateProcess' removal in CollimationAndScattering.cpp

File: ExamplesTutorials/Tutorials/7-CollimationAndScattering/CollimationAndScattering.cpp (hunk 1 above)
Comment (Sam-Tygier) [reply ExamplesTutorials/Tutorials/7-CollimationAndScattering/CollimationAndScattering.cpp:236]: The tracker takes ownership and does the deletion. Happens in the ProcessStepManager destructor.

(On the TODO list for smart pointers :-) )

Comment (scottrowan) [reply ExamplesTutorials/Tutorials/7-CollimationAndScattering/CollimationAndScattering.cpp:236]: All good, quite proficient with smart pointers now, btw, happy to help :)

**Comment by scottrowan** [comment]
All good here, feel free to merge.

**Comment by Sam-Tygier** [comment]
Thanks

**Pull request merged**